readthedocs/core/signals.py

```diff
@@ -33,6 +36,55 @@
 post_collectstatic = Signal()
 
 
+@receiver(email_confirmed)
+def process_email_confirmed(request, email_address, **kwargs):
+    """
+    Steps to take after a users email address is confirmed.
+
+    We currently:
+
+    * Subscribe the user to the mailing list if they set this in their signup.
+    * Add them to a drip campaign for new users in the new user group.
+    """
+
+    # email_address is an object
+    # https://github.com/pennersr/django-allauth/blob/6315e25/allauth/account/models.py#L15
+    user = email_address.user
+    profile = UserProfile.objects.filter(user=user).first()
+    if profile and profile.mailing_list:
```

Comment:
I think we aren't unsubscribing people in this PR still, which we should be doing, but I think we can do it later. 

```suggestion
    if profile and profile.mailing_list:
    # TODO: Unsubscribe users if they unset `mailing_list`.
```